bin/gen

```diff
@@ -52,6 +52,7 @@ gen_rust() {
     rm -rf rust
     gen -g rust -o /work/rust --package-name mzcloud
     (cd rust && rm_standard)
+    (cd rust && cargo fmt)
```

Comment:
The previous line also had a `cd rust`. Will this try to change the directory to `rust/rust`?